src/commands/deploy.ts

```diff
@@ -23,64 +22,40 @@ import { isPathEqual, isSubpath } from '../utils/pathUtils';
 import { getRandomHexString } from "../utils/randomUtils";
 import * as workspaceUtil from '../utils/workspace';
 import { cancelWebsiteValidation, validateWebSite } from '../validateWebSite';
+import { getDefaultWebAppToDeploy } from './getDefaultWebAppToDeploy';
 import { startStreamingLogs } from './startStreamingLogs';
 
 // tslint:disable-next-line:max-func-body-length cyclomatic-complexity
```

Comment:
I like moving `getDefaultWebAppToDeploy` to its own file - does that mean we enable either or both of these rules now? 😄

Reply:
There's still quite a bit of work to do on that front. . .

![image](https://user-images.githubusercontent.com/5290572/58981225-cf03a200-8786-11e9-9bf8-f0460e847f8c.png)

😓

Reply:
Oof